testsuite/testcases/submit_gpu_1x1.yaml

```diff
@@ -0,0 +1,24 @@
+numBatches: 1
+batchSize: 1
+queue: e2e-test-queue
+jobs:
+  - priority: 0
+    namespace: personal-anonymous
+    podSpec:
+      containers:
+        - name: cuda-container
+          image: nvcr.io/nvidia/k8s/cuda-sample:vectoradd-cuda10.2
+          resources:
```

Comment:
I think general good practice would be to specify all resources (cpu, memory) as we do in all other tests